Config/bootstrap.php

```diff
@@ -27,13 +27,13 @@
 
 // Load application configurations
 $conf = array();
-$files = array('application.yml', 'application.local.yml');
+$files = array('application.yml', 'application.local.yml', env('HTTP_HOST') . '.yml', env('HTTP_X_FORWARDED_HOST') . '.yml');
```

Comment:
以下のように `application.` prefix を付けたほうが管理しやすいのではと思った。どうでしょ？

```suggestion
$files = array('application.yml', 'application.local.yml', 'application.' . env('HTTP_HOST') . '.yml', 'application.' . env('HTTP_X_FORWARDED_HOST') . '.yml');
```

Reply:
Config/Environments/xxx.yml みたいな感じにした方がいいかもしれないと思いました。